integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/environment/EnvironmentTestDto.java

```diff
@@ -42,7 +42,6 @@
 import com.sequenceiq.it.cloudbreak.dto.credential.CredentialTestDto;
 import com.sequenceiq.it.cloudbreak.dto.telemetry.TelemetryTestDto;
 import com.sequenceiq.it.cloudbreak.search.Searchable;
-import com.sequenceiq.it.util.TagAdderUtil;
 
 @Prototype
 public class EnvironmentTestDto
```

Comment:
this is not inherited from AbstractTestDto!

Reply:
it is a bit more abstract, but inherited
![image](https://user-images.githubusercontent.com/12979920/86008036-903b2c80-ba18-11ea-870c-edb569aafb40.png)

Reply:
I didn't see the `extends` line below, sorry.